server/data/data-ldap-integration-testing/src/test/java/org/apache/james/user/ldap/DockerLdapRule.java

```diff
@@ -17,39 +17,30 @@
  * under the License.                                           *
```

Comment:
You should not be doing that move. `TestExtensions` are specific to guice, and we should not leak them in other places.

As you don't use it in `data-ldap-integration-test` I bet you could safely move it back to `cassandra-ldap-guice`...